### PR TITLE
🥅 Catch not found error for NFT purchase info & metadata

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: src/mixins/nft.js [hunk 1]
```diff
@@ -320,11 +320,25 @@ export default {
       }
     },
     async updateNFTClassMetadata() {
-      await this.fetchNFTMetadata(this.classId);
+      try {
+        await this.fetchNFTMetadata(this.classId);
+      } catch (error) {
+        if (error.response?.status !== 404) {
+          // eslint-disable-next-line no-console
+          console.error(error);
```
Comment (williamchong) [inline src/mixins/nft.js:328]: please use JSON.stringify since axios error is multiline

File: src/mixins/nft.js [hunk 2]
```diff
@@ -320,11 +320,25 @@ export default {
...
       this.updateDisplayNameList(this.iscnOwner);
     },
     async updateNFTPurchaseInfo() {
-      await this.fetchNFTPurchaseInfo(this.classId);
+      try {
+        await this.fetchNFTPurchaseInfo(this.classId);
+      } catch (error) {
+        if (error.response?.status !== 404) {
+          // eslint-disable-next-line no-console
+          console.error(error);
```
Comment (williamchong) [inline src/mixins/nft.js:339]: please use JSON.stringify since axios error is multiline

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/mixins/nft.js (hunk 2 above)
Comment (nwingt) [reply src/mixins/nft.js]: Updated

File: src/mixins/nft.js (hunk 1 above)
Comment (nwingt) [reply src/mixins/nft.js]: Updated

**Review by williamchong — APPROVED** [review]

**Pull request merged**